packages/backend/src/infrastructure/express/router/user.routes.ts

```diff
@@ -0,0 +1,28 @@
+import express from 'express'
+import { CommonRoutesConfig } from './common.routes.config'
+import userController from '../../../modules/user/user.controller'
+
+const V1 = '/v1'
```

Comment:
That's my bad. We should extract this value to some config file 😄

Reply:
Added to .env as API_VERSION